portfolio.html

```diff
@@ -28,189 +29,38 @@
   </head>
 
   <body>
-    <header>
-      <nav class="topnav">
-        <a class="mylogo" href="#mylogo">My Logo</a>
-        <ul class="nav__links">
-          <li class="line"><a href="#portfolio"></a></li>
-          <li class="line"><a href="#about"></a></li>
-          <li class="line"><a href="#contact"></a></li>
-        </ul>
```

Comment:
- [ ] Notice that the text color here is not exactly black. It's in fact `#344563`. Please make these changes and fix any other sections that have the wrong `font-family` or `font-color`s.

Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/155623198-3c7a0144-6983-4ee7-a638-6b0ef229a1d4.png)|![image](https://user-images.githubusercontent.com/18730106/155623213-55ad7962-6915-454e-8fab-81a79ceaf6bf.png)

Reply:
@MrBrN197 . Nobody is approving the project despite taken effort to implement all changes. Are you people just to find issues or do you have any responsibility to approve or check if changes are implemented?